web-services/handler.js

```diff
@@ -8,9 +8,45 @@ module.exports.ping = async (event, context) => {
       input: event,
     }),
   };
+};
+//--------------------------------------------------
+//This function uses axios to get Trello API boards
+//--------------------------------------------------
+
+const axios = require('axios');
 
+const getData = async () => {
+  try {
+    return await axios.get('https://api.trello.com/1/members/me/boards?', {
+        params: {
+            key: '96e992b35b800ab9db9dc719439519f0',
+            token: '3c50380995f42d4fd2818dc2cdb95aae20c98c1a9231ea037df806ccd4f56898',
+            fields: 'name,id'
+        }
+    });
+  } catch (error) {
+    console.error(error);
+  }
+} 
+//--------------------------------------------------
+//The below function will process the API data and
+//put them back to the Board endpoint 
+//--------------------------------------------------
+module.exports.getBoard = async (event, context) => {
+  const tb = await getData();
```

Comment:
This should be in a try/catch block instead of having the try/catch being in the "getData" function. The reason for this is that you will want to send a different status code (i.e. not a 200) if something goes wrong. The getData could raise an Error which can be trapped by this function for logging.

Reply:
I would like to make sure that I put try/catch for the code like below and removed try/catch in getData function. Is that what you recommended?

try{ 
    var tb = await getData();
  } catch (error){
    console.log();
  }